### PR TITLE
[en]: add hwameistor/resources/volume-snapshot

### PR DESCRIPTION
**Pull request by WANG0608GitHub**
Update after PR #5376

### CODE REVIEW

**Comment by WANG0608GitHub** [comment]
see preview:
https://deploy-preview-5400--daocloud-docs.netlify.app/en/storage/hwameistor/resources/volume-snapshot

File: docs/en/docs/storage/hwameistor/resources/volume-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+# Volume Snapshot
+
+In HwameiStor, users can create snapshots of data volumes （VolumeSnapshot）, and restore or roll back
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/volume-snapshot.md:3]:
```suggestion
In HwameiStor, users can create snapshots of data volumes (VolumeSnapshot), and restore or roll back
```
Do not use Chinese chars

File: docs/en/docs/storage/hwameistor/resources/volume-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+based on these snapshots.
+
+!!! note
+
+    Currently, only LVM data volumes without high availability support snapshot creation.
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/volume-snapshot.md:8]:
```suggestion
    Currently, only LVM volumes without high availability support snapshot creation.
```

File: docs/en/docs/storage/hwameistor/resources/volume-snapshot.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+After creating a VolumeSnapshot, you can check the Hwameistor LocalvolumeSnapshot using the following command.
+
+```console
+$ kubectl get lvs
+NAME                                               CAPACITY     SOURCEVOLUME                               STATE   MERGING   INVALID   AGE
+snapcontent-0fc17697-68ea-49ce-8e4c-7a791e315110   1073741824   pvc-967baffd-ce10-4739-b996-87c9ed24e635   Ready                       5m31s
+```
+
+- CAPACITY: The capacity size of the snapshot
+- SourceVOLUME: The source volume name of the snapshot
```
Comment (windsonsea) [inline docs/en/docs/storage/hwameistor/resources/volume-snapshot.md:71]:
```suggestion
- SOURCEVOLUME: The source volume name of the snapshot
```
这指的是 line 66 命令行中的字段

File: docs/zh/docs/storage/hwameistor/resources/volume-snapshot.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ snapcontent-0fc17697-68ea-49ce-8e4c-7a791e315110   1073741824   pvc-967baffd-ce1
 ```
 
 - CAPACITY：快照的容量大小
-- SOURCEVOLUME：快照的源卷名称
+- SourceVOLUME：快照的源卷名称
```
Comment (windsonsea) [inline docs/zh/docs/storage/hwameistor/resources/volume-snapshot.md:68]:
```suggestion
- SOURCEVOLUME：快照的源卷名称
```

**Review by windsonsea — APPROVED** [review]

**Pull request merged**